p2p/src/peers.rs

```diff
@@ -74,7 +76,9 @@ pub async fn run(
                 do_tick(expiration, &node_key, addresses.iter().cloned().collect(), &svc).await
             }
             Some(m) = messages.recv() => {
-                handle_message(node_key.id(), m, &mut addresses,&svc).await
+                if handle_message(node_key.id(), m, &mut addresses,&svc).await{
```

Comment:
interesting, this formatting seems weird without a space for `addresses,&svc`. I'm fine with it since clippy/ci approve 🤷‍♂️